src/Illuminate/Cache/Repository.php

```diff
@@ -573,18 +575,22 @@ public function offsetUnset($key)
     /**
      * Calculate the number of minutes with the given duration.
      *
-     * @param  \DateTimeInterface|\DateInterval|float|int  $duration
+     * @param  \DateTimeInterface|\DateInterval|float|int|null  $minutes
      * @return float|int|null
      */
-    protected function getMinutes($duration)
+    protected function getMinutes($minutes)
     {
-        $duration = $this->parseDateInterval($duration);
+        if ($minutes === null) {
```

Comment:
Any reason for not `if (is_null($minutes)) {`?

Reply:
`is_null()` is only useful in callbacks: a direct comparison makes things faster (although not relevant until you do it a gazillion times), and more clear to downstream static introspection (AST directly tells you the result, instead of having to look up the `is_null()` definition and writing custom assumptions in the static analysis tooling)